gotocompany/compass/v1beta1/service.proto

```diff
@@ -816,6 +832,52 @@ message SearchAssetsRequest {
   }];
 }
 
+
+message GroupAssetsRequest {
+
+  string groupby = 1 [
```

Comment:
Why not keep this as a repeated string if we want to allow for multiple fields?

Reply:
Also, how does changing it to repeated string translate while calling the endpoint from HTTP?

Reply:
If repeated http request would be like this : /groupassets?groupby=type&groupby=service&include_fields=name
If not repeated it would be like this: /groupassets?groupby=type,service&include_fields=name.
Second one would be better as in first case we are reducing the size of request and that is the common way for http requests.

Reply:
Shouldn't we optimise for the API contract in proto definitions? Having a string groupby necessitates a comment explaining it can be a CSV. Having a repeated string is much more clear. 
Reducing the size of request by a minuscule amount cannot be considered better IMHO. The way of specifying multiple values for a key in query string is standard and well-understood.

Reply:
The group by fields will by default be part of the response, for other fields the user will have to mention in included fields, since the asset holds a lot of keys, this could result in a huge query string.

Reply:
Can you please respond to the points I raised. I am still not convinced there will be a considerable difference in query string size in practice. And even if there were a size difference, why is that a concern?